cirq-google/cirq_google/api/v2/program.proto

```diff
@@ -424,4 +425,13 @@ message InternalGate{
   string module = 2;  // Gate module. 
   int32 num_qubits = 3;  // Number of qubits. Required during deserialization.
   map<string, Arg> gate_args = 4;  // Gate args.
+}
+
+message CouplerPulseGate{
+  optional FloatArg hold_time_ps = 1;
```

Comment:
Please add line comments.  ps=pico seconds?  Do we foresee a need for sub-nanosecond timing?

Reply:
added. initially I was using ns, but when I wrote a test that used ps I got precision error. the test input was `cirq.Duration(picos=1)` and when the object got translated it had `cirq.Duration(nanos=1.0000000..1e-3)`. so I'm using ps to avoid that.l